targets/three/src/index.ts

```diff
@@ -19,7 +19,6 @@ addEffect(() => {
 })
 
 const host = createHost(primitives, {
-  // @ts-expect-error r3f related
   applyAnimatedValues: applyProps,
 })
```

Comment:
This should probably be a proxy since it is valid to `extend` the JSX interface beyond what the `THREE` namespace contains.